javascript/javascript2/week2/hyf-bay/main.js

```diff
@@ -0,0 +1,82 @@
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+let resultedProducts = products;
+
+console.log(productsUl);
+
+function renderProducts(products) {
+  products.forEach(product => {
+    const li = document.createElement('li');
+    let shipsToHTML = '';
+    product.shipsTo.forEach(country => shipsToHTML += `<li>${country}</li>`);
```

Comment:
`product.shipsTo` is undefined there is not property call shipsTo in your `product object`.  You will get error, I suggest you to remove line 12 and 13.